schema/types/mutation.js

```diff
@@ -40,5 +40,17 @@ module.exports = `
             date: String!
             order: Int!
         ): TimesheetRecord @auth(companyIdField: "companyId", projectIdField: "projectId")
+        
+         updateTimesheetRecord(
+            timesheetRecordId: ID!
+            projectId: ID!
+            hours: Float!
+            comment: String!
+            order: Int!
+        ): TimesheetRecord @auth(companyIdField: "companyId", projectIdField: "projectId")
```

Comment:
Auth checks fields among those passed to the mutation, companyId either needs to be passed to updateTimesheetRecord or it needs to be removed from the auth directive

Reply:
fix